Catalog/DataAccessors/Comments.cs

```
@@ -47,38 +48,33 @@ public async Task<int> Delete(int id)
             }
         }
 
-        public async Task<int> Edit(Comment comment)
+        public async Task<int> Edit(int Id, string Message)
         {
             using (var context = _sql.Context)
             {
                 var query = @"
                         UPDATE [Comments]
                         SET";
-                if (!string.IsNullOrWhiteSpace(comment.Author))
-                {
-                    query += "[Author] = @author";
-                }
-                if (!string.IsNullOrWhiteSpace(comment.Message))
+                if (!string.IsNullOrWhiteSpace(Message))
```

I disagree with this one. There should be no cases when we are not having a message passed as the parameter. Otherwise, it has no sense.

- [ ] id and message validation should be performed before query assembling
- [ ] you should throw the exception(s) (like ArgumentNullException or other you think are relevant here) when either of the parameters is null or empty.